weave-js/src/components/Panel2/ChildPanelExportReport/customSelectComponents.tsx

```diff
@@ -0,0 +1,72 @@
+import {
+  components as selectComponents,
+  OptionProps,
+  SingleValueProps,
+} from 'react-select';
+import {Icon} from '../../Icon';
+import React from 'react';
+import {EntityOption, ReportOption, formatUpdatedBy} from './utils';
+import TimeAgo from 'react-timeago';
+
+export const customEntitySelectComps = {
+  SingleValue: ({
+    children,
+    ...props
+  }: SingleValueProps<EntityOption, false>) => (
+    <selectComponents.SingleValue
+      {...props}
+      className="flex items-center gap-8">
+      <Icon
+        className="shrink-0 grow-0 "
+        name={props.data?.isTeam ? 'users-team' : 'user-profile-personal'}
+      />
+      <p className="overflow-hidden text-ellipsis">{children}</p>
+    </selectComponents.SingleValue>
+  ),
+  Option: ({children, ...props}: OptionProps<EntityOption, false>) => {
+    return (
+      <selectComponents.Option {...props} className="flex items-center  gap-8">
+        <Icon
+          name={props.data?.isTeam ? 'users-team' : 'user-profile-personal'}
+        />
+        <span>{children}</span>
+      </selectComponents.Option>
+    );
+  },
+};
+
+export const customReportSelectComps = {
+  Option: ({children, ...props}: OptionProps<ReportOption, false>) => {
+    const optionData: ReportOption = props.data;
+    return (
+      <selectComponents.Option {...props} className="flex items-center">
+        <div className="flex grow">
+          <Icon name="report" className="shrink-0 grow-0 pt-4" />
+          <p className="mx-8 flex grow flex-col gap-4">
+            <span>{children}</span>
+            <span className="text-sm text-moon-500">
+              {optionData.projectName}
+            </span>
+          </p>
+        </div>
+        {optionData?.updatedAt != null && (
+          <p className="shrink-0 text-sm text-moon-500">
+            <TimeAgo
+              aria-label="datetime of last report edit"
```

Comment:
**issue (accessibility):** `aria-label` is only meant for interactive/focusable elements. This time stamp should not have a label

Reply:
Removed - i started reading about aria labels and def need to do some more research in this area